packages/api/src/env.js

```diff
@@ -8,14 +8,12 @@ import { DBClient } from '@web3-storage/db'
  * @param {Env} env
  */
 export function envAll (_, env) {
-  const getConfig = s => env[s] || globalThis[s]
-
-  env.magic = new Magic(getConfig('MAGIC_SECRET_KEY'))
+  env.magic = new Magic(env.MAGIC_SECRET_KEY || MAGIC_SECRET_KEY)
 
   env.db = new DBClient({
-    endpoint: getConfig('FAUNA_ENDPOINT'),
-    token: getConfig('FAUNA_KEY')
+    endpoint: env.FAUNA_ENDPOINT || FAUNA_ENDPOINT,
+    token: env.FAUNA_KEY || FAUNA_KEY
   })
 
-  env.SALT = getConfig('SALT')
+  env.SALT = env.SALT || SALT
```

Comment:
esbuild was thinking these globals were "dead code" and removing them.

Reply:
yes, i'm not super comfortable with their being an alternative bundling chain that is only used in the playwright-test --sw flow.

Reply:
Me either. @hugomrdias is there a way we can use our own build in the sw? There's no code that playwright has to add to it...right?